examples/text_input_aligned/src/main.rs

```diff
@@ -0,0 +1,79 @@
+use iced::{
+    text_input, Application, Column, Command, Container, Element,
+    HorizontalAlignment, Length, Settings, Text, TextInput,
+};
+
+struct State {
+    message: String,
+
+    // Local Element States
+    input_state: text_input::State,
+}
+
+#[derive(Debug, Clone)]
+enum Event {
+    OnChange(String),
+}
+
+impl Application for State {
+    type Executor = iced::executor::Default;
+    type Message = Event;
+    type Flags = ();
+
+    fn new(_flags: ()) -> (Self, Command<Event>) {
+        (
+            State {
+                message: "Hello World".to_owned(),
+                input_state: text_input::State::focused(),
+            },
+            Command::none(),
+        )
+    }
+
+    fn title(&self) -> String {
+        "Hello World Example".to_owned()
+    }
+
+    fn update(&mut self, event: Event) -> Command<Event> {
+        match event {
+            Event::OnChange(message) => {
+                self.message = message;
+            }
+        };
+
+        Command::none()
+    }
+
+    fn view(&mut self) -> Element<Event> {
+        let input = TextInput::new(
+            &mut self.input_state,
+            "Type a Message",
+            &self.message,
+            Event::OnChange,
+        )
+        .padding(15)
+        .size(30)
+        .horizontal_alignment(HorizontalAlignment::Left);
+
+        let text = Text::new(self.message.to_string())
+            .width(Length::Fill)
+            .size(100)
+            .color([0.5, 0.5, 0.5])
+            .horizontal_alignment(HorizontalAlignment::Center);
+
+        let column = Column::new().push(input).push(text);
+
+        Container::new(column)
+            .padding(100)
+            .width(Length::Fill)
+            .height(Length::Fill)
+            .into()
+    }
+}
+
+fn main() {
+    log4rs::init_file("examples/hello_world/log4rs.yml", Default::default())
+        .unwrap();
+
+    State::run(Settings::default());
+}
```

Comment:
Maybe it'd be interesting to merge this example in the `tour`, as it's only showcasing a single method of a `TextInput`. For now, it's very useful for testing, though!